react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.numberOfClicks = 0;
```

Comment:
You should still have a state, it should just contain the number of clicks. Sorry if I wasn't clear.